packages/graph-explorer/src/core/StateProvider/displayAttribute.ts

```diff
@@ -0,0 +1,52 @@
+import { Vertex, Edge } from "@/@types/entities";
+import { AttributeConfig } from "../ConfigurationProvider";
```

Comment:
Nit: IntelliJ is saying this could be `import { AttributeConfig } from "@/core";`